website/templates/emails/notify_base.mako

```diff
@@ -43,7 +43,7 @@
                 <tbody>
                     <tr>
                         <td style="border-collapse: collapse;">
-                            <p class="small text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or user settings.</p>
+                            <p class="small text-center" style="text-align: center;font-size: 12px;">To change your email notification preferences, visit your project or <a href="https://osf.io/settings/notifications/">user settings</a>.</p>
```

Comment:
This url will break on staging. Instead of hard-coding the `https://osf.io/` can you use `settings.DOMAIN` (from `website.settings`)?